koku/masu/processor/ocp/ocp_cloud_parquet_summary_updater.py

```diff
@@ -77,6 +78,7 @@ def update_aws_summary_tables(self, openshift_provider_uuid, aws_provider_uuid,
                     current_ocp_report_period_id,
                     current_aws_bill_id,
                     markup_value,
+                    distribution,
```

Comment:
In this case, we want to check the distribution on the associated OpenShift cluster's cost model, *not* the cost model for the cloud source. We still get the markup from the cloud source, though.